JWTManipulator9000/README.md

```diff
@@ -0,0 +1,28 @@
+# JWTManipulator9000
+A WASM filter made to manipulate JWT token headers and payloads. Works best with the Meshery Project :)
+
+Sample configuration to be passed:
+```json
+{
+  "add_header": [
+    ["test1","test1"],
```

Comment:
Does each set represent a key/value pair?

`["test1","test1"],`
 e.g. test1 is the key and test1 is the value?

Reply:
I completely agree, but at the time i remembered this worked well, so I left it. I need to revise my serde deserialize notations xD